corehq/apps/export/static/export/js/export_list.js

```diff
@@ -140,11 +141,17 @@ hqDefine("export/js/export_list", [
                         var exportType = utils.capitalize(model.exportType());
                         googleAnalytics.track.event(exportType + " Exports", "Update Saved Export", "Saved");
                         model.pollProgressBar();
+                    } else {
+                        self.handleExportError(data);
                     }
                 },
             });
         };
 
+        self.handleExportError = function (data) {
+            self.prepareExportError(gettext(data.error));
```

Comment:
You can remove the `gettext` - it can't translate a variable, only static strings, and you're already translating the error message in python.